.github/workflows/ci.yml

```diff
@@ -1582,6 +1586,9 @@ jobs:
             xtensa-sample_controller_zephyr-elf)
               PLATFORM_ARGS+="-p qemu_xtensa "
               ;;
+            xtensa-sample_controller32_zephyr-elf)
+              PLATFORM_ARGS+="-p qemu_xtensa/sample_controller32/mpu "
+              ;;
```

Comment:
This needs to be disabled until the board support is merged on the Zephyr side; otherwise, CI will fail.

Reply:
Done